packages/ui5-middleware-cfdestination/package.json

```diff
@@ -25,7 +25,9 @@
     "@sap/approuter": "^13.1.0",
     "@ui5/logger": "^2.0.1",
     "content-type": "^1.0.5",
-    "node-fetch": "^3.3.0"
+    "express-http-proxy": "^1.6.3",
```

Comment:
ˋcontent-typeˋ missing?

Reply:
`content-type` is listed in line 27 above - was already a dependency before.